internal/workflow_testsuite.go

```diff
@@ -83,6 +84,15 @@ type (
 		runFn        func(args mock.Arguments)
 		waitDuration func() time.Duration
 	}
+
+	// TestUpdateCallback is a basic implementation of the UpdateCallbacks interface for testing purposes.
+	// Tests are welcome to implement their own version of this interface if they need to test more complex
+	// update logic. This is a simple implementation to make testing basic Workflow Updates easier.
+	TestUpdateCallback struct {
```

Comment:
The issue was intended to improve the UX for users so we would need to expose this https://github.com/temporalio/sdk-go/blob/master/testsuite/testsuite.go#L35

Reply:
oops! didn't realize this was needed. Thanks for pointing it out